docs/pages/guides/server-components.mdx

```diff
@@ -0,0 +1,400 @@
+
+As of SDK 52, there's no strategy to migrate existing projects, we recommend starting from a new project.
+
+Enable the flag in the project Expo config:
+
+```json app.json
+{
+  "expo": {
+    "experiments": {
+      "reactServerComponents": true
+    }
+  }
+}
+```
+
+Ensure the entry module is `expo-router/entry` (default).
+
+Create an initial route `app/index.tsx`:
+
+```tsx app/index.tsx
+// This is a server component.
+import { Text } from 'react-native';
+
+export default function Index() {
+  return <Text>Hello, world!</Text>;
+}
+```
+
+## Moving between routes
+
+You can navigate between routes by using the `<Link />` API from Expo Router (not all props are supported yet).
+
+```tsx app/index.tsx
+import { Link } from 'expo-router';
+
+export default function Index() {
+  return (
+    <Link href="/second">
+      <Text>Hello, world!</Text>
+    </Link>
+  );
+}
+```
+
+## Layout Routes
+
+> Complex built-in layouts such as Stack, and Tabs are not supported yet. They'll fallback to a `<Slot />`.
+
+Layout routes (`app/_layout.js`) can be used to wrap all routes in a common layout.
+
+```tsx app/_layout.tsx
+import { View } from 'react-native';
+import { Slot } from 'expo-router';
+
+export default function Layout() {
+  return (
+    <View>
+      <Slot />
+    </View>
+  );
+}
+```
+
+The `Slot` component is used to render the current route.
+
+## Server Components
+
+Server Components run in the server, meaning they can access Node.js built-ins and server APIs. They can also use async components.
+
+Consider the following component which fetches data and renders it:
+
+```tsx app/index.tsx
+import { Image, Text, View } from 'react-native';
+
+export async function Pokemon() {
+  const res = await fetch('https://pokeapi.co/api/v2/pokemon/2');
+  const json = await res.json();
+  return (
+    <View style={{ padding: 8, borderWidth: 1 }}>
+      <Text style={{ fontWeight: 'bold', fontSize: 24 }}>{json.name}</Text>
+      <Image source={{ uri: json.sprites.front_default }} style={{ width: 100, height: 100 }} />
+
+      {json.abilities.map(ability => (
+        <Text key={ability.ability.name}>- {ability.ability.name}</Text>
+      ))}
+    </View>
+  );
+}
+```
+
+- You cannot use hooks like `useState`, `useEffect`, or `useContext` in Server Components.
+- You cannot use browser or native APIs in Server Components.
+- `"use server"` is not meant to mark a file as a server component. It's used to mark a file as having React Server Actions.
+- Server components have access to all environment variables as they run securely off the client.
+
+## Client Components
+
+Server Components run in the server and have no access to native APIs or React Context. To use these features, you can create a Client Component. Client Components are created by marking files with the `"use client"` directive at the top.
```

Comment:
```suggestion
Since Server Components cannot access native APIs or React Context, you can create a Client Component to use these features. Client Components are created by marking files with the `"use client"` directive at the top.
```

We've already explained where Server Components run in the previous section, so let's not repeat that here.